modules/monitoring/files/usr/lib/nagios/plugins/check_reboots_required

```diff
@@ -3,9 +3,11 @@
 def create_machines_hash
   machines_names_hash = Hash.new
 
-  `/usr/local/bin/govuk_node_list --with-puppet-class`.each_line do |machine|
-    hostname, name = machine.chomp.split(":")
-    machines_names_hash[hostname] = name
+  if File.file?("/etc/facter/facts.d/aws_environment.txt")
```

Comment:
`File.exist?(...` might be a bit more meaningful here.

Also, it's a bit obscure putting this conditional within `create_machines_hash`, but refactoring to avoid this might be a bit too much. Maybe just simplify the conditional to make it more explicit that the function just returns an empty hash in the non-AWS case, for example:

```ruby
def create_machines_hash
  return {} unless File.exist? '/etc/facter/facts.d/aws_environment.txt'

  ...
```

Reply:
done, thanks!